src/stories/SectionRemoveSearchFilter.stories.js

```diff
@@ -0,0 +1,65 @@
+import SectionRemoveSearchFilter from '@/lib-components/SectionRemoveSearchFilter'
+
+export default {
+  title: 'SEARCH / Section Remove Search Filter',
+  component: SectionRemoveSearchFilter,
+}
+const mockRework = [
```

Comment:
@pghorpade Looks like a duplicate mock setup.

Reply:
yes I will remove it